### PR TITLE
refactor(cxl-lumo-styles): update cxl-lumo-styles and storybook

### PR DESCRIPTION
**Pull request by freudFlintstone**
https://app.clickup.com/t/861mhf04m

### CODE REVIEW

File: packages/cxl-lumo-styles/scss/_mixins.scss [hunk 1]
```diff
@@ -43,3 +43,65 @@
...
+  font-weight: 300;
+}
+
+@mixin regular {
+  font-weight: 400;
+}
+
+@mixin bold {
+  font-weight: 900;
+}
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/_mixins.scss]: Naming them `font-weight-light` etc. would be better

File: packages/cxl-lumo-styles/scss/_mixins.scss [hunk 2]
```diff
@@ -43,3 +43,65 @@
...
+
+@mixin italic {
+  font-style: italic;
+}
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/_mixins.scss]: It's a wrapper for regular CSS style. Could you explain how this is better then just doing `font-style: italic;` everywhere?

File: packages/cxl-lumo-styles/scss/_mixins.scss [hunk 3]
```diff
@@ -43,3 +43,65 @@
...
+
+@mixin label {
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/_mixins.scss]: What kind of label is it? Where should we use it? Please add comment

File: packages/cxl-lumo-styles/scss/_mixins.scss [hunk 4]
```diff
@@ -43,3 +43,65 @@
...
+  line-height: var(--cxl-lumo-line-height-s); // closest
+}
+
+@mixin component-text {
+  @extend regular;
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--cxl-lumo-line-height-m);
+}
+
+@mixin body-text {
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/_mixins.scss]: These look similar and it's no clear for me where should we use each of these. Please add comments.

File: packages/cxl-lumo-styles/scss/_mixins.scss [hunk 5]
```diff
@@ -43,3 +43,65 @@
...
+  @extend regular;
+  font-family: var(cxl-lumo-font-secondary);
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/_mixins.scss]: `--` missing?

File: packages/cxl-lumo-styles/scss/themes/vaadin-button.scss [hunk 6]
```diff
@@ -0,0 +1,10 @@
+:host([theme~="large"].cxl-cta) {
+  --lumo-button-size: 3em;
+  cursor: pointer;
+  padding: 1.25em 3.5em 1.25em 3em;
+  font-size: var(var-lumo-font-size-l) !important;
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/themes/vaadin-button.scss]: `--` missing?

File: packages/storybook/cxl-lumo-styles/typography.stories.js [hunk 7]
```diff
@@ -108,7 +107,7 @@ export const CXLComExamples = () => html`
       <h3>Master the most in-demand marketing skills with our online courses created by marketers with a proven record of driving real growth.</h3>
       <vaadin-button id="cxl-onboarding" class="cxl-cta" role="button" theme="primary large">
         Sign up now
-        <vaadin-icon slot="suffix" icon="lumo:arrow-right"></vaadin-icon>
+        <!-- <vaadin-icon slot="suffix" icon="lumo:arrow-right"></vaadin-icon> -->
```
Comment (pawelkmpt) [inline packages/storybook/cxl-lumo-styles/typography.stories.js:110]: Why not just delete it?

File: packages/storybook/cxl-lumo-styles/typography.stories.js [hunk 8]
```diff
@@ -174,7 +173,7 @@ export const CXLComExamples = () => html`
     </div>
     <vaadin-button id="cxl-onboarding" class="cxl-cta wide" role="button" theme="primary large">
       Sign up now
-      <vaadin-icon slot="suffix" icon="lumo:arrow-right"></vaadin-icon>
+      <!-- <vaadin-icon slot="suffix" icon="lumo:arrow-right"></vaadin-icon> -->
```
Comment (pawelkmpt) [inline packages/storybook/cxl-lumo-styles/typography.stories.js:176]: Why not just delete it?

File: packages/cxl-lumo-styles/scss/_mixins.scss [hunk 9]
```diff
@@ -43,65 +43,3 @@
     max-width: $max-width;
   }
 }
-
-/*
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/_mixins.scss]: And here I see you deleted mixins. Could you add some context?

File: packages/cxl-ui/scss/cxl-dashboard-header.scss [hunk 10]
```diff
@@ -45,33 +45,10 @@
...
-      border-radius: 100%;
-    }
-  }
-
   .title {
     margin: 0;
-    font-size: var(--lumo-font-size-xxxl);
-    line-height: 1.4;
+    // font-size: var(--lumo-font-size-xxxl);
+    // line-height: 1.4;
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-dashboard-header.scss:51]: just delete

**Review by pawelkmpt — COMMENTED** [review]
@freudFlintstone quick glance review on this draft code. I'll go deeper once PR is ready for review

**Review by lkraav — CHANGES_REQUESTED** [review]

**Comment by freudFlintstone** [comment]
Task linked: [CU-861n2cqe7 CXL Style sync](https://app.clickup.com/t/861n2cqe7)

File: .gitignore [hunk 11]
```diff
@@ -5,3 +5,4 @@ packages/**/pkg
 coverage
 tmp
 test/visual/screenshots
+.idea
```
[the remark this answers is not in the document]
Comment (freudFlintstone) [reply .gitignore]: Looks like it's been there for a while. I'm fixing it in this branch, but we should have specific tasks and PRs for maintenance chores.  
https://github.com/conversionxl/aybolit/commit/639ab2d78171ed2f9f7b8fdf5ba1c9d81e6145ff

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
@freudFlintstone `yarn build` fails. As you can see, GH couldn't do it either.

Screenshot 2023-11-22 at 11 15 34

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**